### PR TITLE
first pass & add randomchoice function

### PR DESCRIPTION
**Pull request by jihan1218**

### CODE REVIEW

File: pysc2/pysc2/lib/pythonUtils.js [hunk 1]
```diff
@@ -178,6 +178,10 @@ function NotImplementedError(message) {
   this.message = str;
 }
 
+function randomChoice(arr) {
```
Comment (arcman7) [inline pysc2/pysc2/lib/pythonUtils.js:181]: add functionality for when arr is an integer

File: pysc2/pysc2/agents/random_agent.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const path = require('path'); //eslint-disable-line
+const base_agent = require(path.resolve(__dirname, './base_agent.js'))
+const actions = require(path.resolve(__dirname, '..', 'lib', './action.js'))
+const pythonUtils = require(path.resolve(__dirname, '..', 'lib', './.pythonUtils.js'))
+const randomChoice = pythonUtils
```
Comment (arcman7) [inline pysc2/pysc2/agents/random_agent.js:5]: const { randomChoice } = pythonUtils

if you're curious about this syntax read about object unpacking (destructuring) in javascript
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Comment (arcman7) [reply pysc2/pysc2/agents/random_agent.js:5]: still gotta fix this

File: pysc2/pysc2/agents/scripted_agent.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+const RAW_FUNCTIONS = actions.RAW_FUNCTIONS
+
+const _PLAYER_SELF = features.PlayerRelative.SELF
+const _PLAYER_NEUTRAL = features.PlayerRelative.NEUTRAL
+const _PLAYER_ENEMY = features.PlayerRelative.ENEMY
+
+function _xy_locs(mask) {
+// Mask should be a set of bools from comparison with a feature layer.//
+  const thresh = mask.greater([0]).asType('bool')
+  const coord = numpy.where(thresh)
```
Comment (arcman7) [inline pysc2/pysc2/agents/scripted_agent.js:17]: does this actually work?

File: pysc2/pysc2/agents/random_agent.js (hunk 2 above)
Comment (jihan1218) [reply pysc2/pysc2/agents/random_agent.js:5]: It has been fixed.

File: pysc2/pysc2/agents/scripted_agent.js (hunk 3 above)
Comment (jihan1218) [reply pysc2/pysc2/agents/scripted_agent.js:17]: This has been changed.

**Pull request merged**